frontend/src/test/color-contrast.test.ts

```
@@ -0,0 +1,204 @@
+
+// WCAG 2.1 Level AA contrast ratios
+const WCAG_AA_NORMAL = 4.5; // Normal text
+const WCAG_AA_LARGE = 3.0;  // Large text (18pt+ or 14pt+ bold)
+
+/**
+ * Convert hex color to RGB
+ */
+function hexToRgb(hex: string): { r: number; g: number; b: number } {
+  const cleanHex = hex.replace('#', '');
+  return {
+    r: parseInt(cleanHex.substr(0, 2), 16),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)